layouts/community/page.js

```
@@ -33,23 +25,55 @@ export default function CommunityPage() {
+                          px: 2,
+                          py: 1,
+                          borderRadius: 20,
+                          fontSize: 1,
+                          fontWeight: 600,
+                          border: "1px solid #E9ECEF"
+                        }}
+                      >
+                        {skill}
+                      </Text>
+                    ))}
+                  </Flex>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)